lib/audit_log/audit_log/formatter.rb

```diff
@@ -56,12 +56,18 @@ def sidekiq_current_ctx
               Sidekiq::Context.current
             else
               # versions up to 6.0.0
-              Thread.current[:sidekiq_context]
+              parse_sidekiq_ctx(Thread.current[:sidekiq_context])
             end
-          rescue NoMethodError
+          rescue StandardError
```

Comment:
:see_no_evil: not a great idea. What kind of errors are you expecting?

Reply:
IMO we should at least log (debug level for local devel) what happened so it's not completely swallowed.